debian/appscale_install_functions.sh

```diff
@@ -539,6 +539,19 @@ installapiclient()
     pipwrapper google-api-python-client
 }
 
+installpycapnp()
+{
+    pipwrapper pycapnp
+}
+
+preplogserver()
+{
+    LOGSERVER_DIR="/opt/appscale/logserver"
+    mkdir -p ${LOGSERVER_DIR}
+    FILE_SRC="$APPSCALE_HOME_RUNTIME/LogService/logging.capnp"
+    FILE_DEST="$APPSCALE_HOME_RUNTIME/AppServer/google/appengine/api/logservice/logging.capnp"
+    cp ${FILE_SRC} ${FILE_DEST}
```

Comment:
@FastGeert  here is where the symlink issue is addressed. The rsync on the server has issues with the symlink and the dashboard never comes up because it can't find the logging.capnp file. I had this problem on a cloud deployment and on a single node cluster in vagrant.